lib/capybara-screenshot/s3_saver.rb

```diff
@@ -5,10 +5,11 @@ module Screenshot
     class S3Saver
       DEFAULT_REGION = 'us-east-1'
 
-      def initialize(saver, s3_client, bucket_name)
+      def initialize(saver, s3_client, bucket_name, key_prefix: nil)
```

Comment:
This syntax is not compatible with older versions of Ruby. If I recall, we are not using named args like this anywhere else, so would prefer for not to remain consistent by using `options = {}`